packages/viewer/src/lib/scenes/__tests__/scene.spec.ts

```diff
@@ -269,7 +269,7 @@ describe(Scene, () => {
               {
                 changeMaterial: {
                   material: {
-                    d: 100,
+                    d: 255,
```

Comment:
Did you discuss this with Dan at all?  This is somewhat complicated because we've already released this as a public API.  

To avoid potential backward compatibility issues, we've discussed changing the range to `[0,1]` just for opacity and then during the conversion on the server-side, take anything larger than `1.0` to mean fully opaque.

Reply:
I did not - just knew we had that ticket in the backlog to update these defaults. For the time being I can certainly revert this change